nss-cohort-36/keahua-arboretum-net-surfers
# Sp print report

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-print-report`
4.  run the program and add a few biomes, check that you can name each one.
5. try to display facility report on main menu, check that you get a list of each biome type, with each biome's name

# Checklist:
